src/solver/main.jl

```diff
@@ -0,0 +1,293 @@
+# This file uses Groebner, so it needs to be added in the future.
+
+"""
+    symbolic_solve(expr, x; repeated=false)
```

Comment:
I still prefer "multiplicites" over "repeated", even if it is longer.
Maple uses "dropmultiplicity" in https://www.maplesoft.com/support/help/maple/view.aspx?path=solve%2Fdetails

Reply:
Maybe we also ought to use "dropmultiplicity". It is then very clear what it does.